README.md

```diff
@@ -29,4 +29,13 @@ jupyter nbextension     install --py --sys-prefix nbrsessionproxy
 jupyter nbextension     enable  --py --sys-prefix nbrsessionproxy
 ```
 
+For JupyterLab first clone this repository to a known location and
+install from the directory.
+```
+git clone https://github.com/jupyterhub/nbserverproxy /opt/nbserverproxy
+pip install -e /opt/nbserverproxy
+juptyer serverextension enable --py nbrsessionproxy
+jupyter labextension link /opt/nbrsessionproxy/jupyterlab-rsessionproxy
```

Comment:
I agree, but wasn't sure if I should "own" the npm repo or let one of the original maintainers of this repository "own" the npm keys.